client/lib/cart/store/cart-analytics.js

```diff
@@ -17,6 +17,8 @@ function recordEvents( previousCart, nextCart ) {
 
 function recordAddEvent( cartItem ) {
 	analytics.tracks.recordEvent( 'calypso_cart_product_add', cartItem );
+	analytics.ga.recordEvent( 'Checkout', 'calypso_cart_product_add',
+		`${ cartItem.product_id } ${ cartItem.product_slug }` );
```

Comment:
Actually @lamosty let's pass through `product_slug` as the value arg, and pass in `product_id` as an integer in the value arg (4th param)